include/dlpack/dlpack.h

```diff
@@ -64,14 +64,16 @@ typedef enum {
 } DLDeviceType;
 
 /*!
- * \brief A Device context for Tensor and operator.
+ * \brief A Device for Tensor and operator.
  */
 typedef struct {
   /*! \brief The device type used in the device. */
   DLDeviceType device_type;
   /*! \brief The device index */
   int device_id;
-} DLContext;
+} DLDevice;
+
+typedef DLDevice DLContext;
```

Comment:
Please add a comment about the alias and indicating that we will consider remove it in the next release

Reply:
added the comment.